requirements/base.txt

```diff
@@ -12,10 +12,10 @@ django-pagedown==0.1.1
 django-reversion==1.10.1
 django-selectable==0.9.0
 django-taggit==0.18.0
-django-treebeard==4.0
+git+https://github.com/fladi/django-treebeard@cd4b13d2d58c0c514d1bb4194de9ebb2e562cd77
```

Comment:
Looks good, though I think I may wait a week or so to see if the PR gets accepted and a new version is released; new versions are fairly frequent w/ this library...